src/common/low_precision_transformations/CMakeLists.txt

```
@@ -16,7 +16,7 @@ source_group("src" FILES ${LIBRARY_SRC})
 source_group("include" FILES ${PUBLIC_HEADERS})
 
 # Create library
-if(CMAKE_CXX_STANDARD GREATER_EQUAL 20)
+if(CMAKE_COMPILER_IS_GNUCXX AND CMAKE_CXX_STANDARD GREATER_EQUAL 20)
```

```suggestion
if((CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG) AND CMAKE_CXX_STANDARD GREATER_EQUAL 20)
```
Clang is not `GNUCXX` and should be handled separately.

---

Clang doesn't report it, so I've reduce it to GNU.

---

Updated